train/uft/tuning.py

```diff
@@ -50,7 +50,6 @@ def get_params(trial):
     discriminator_adam_lr = trial.suggest_float('discriminator_adam_lr', 1e-6, 1e-2, log = True)
     denoiser_loss_b1 = trial.suggest_float('denoiser_loss_b1', 0.1, 0.9)
     denoiser_loss_b2 = 1 - denoiser_loss_b1
-    print_loss_interval = trial.suggest_int('print_loss_interval', 50, 500)
     denoiser_adversarial_loss_clip_max = trial.suggest_float('denoiser_adversarial_loss_clip_max', 0.0, 1.0)
     denoiser_adversarial_loss_clip_min = trial.suggest_float('denoiser_adversarial_loss_clip_min', 0.0, 1.0)
     calc_eval_loss_interval = trial.suggest_int("calc_eval_loss_interval", 50, 500)
```

Comment:
this doesn't need to be a parameter either, as it's simply used for recording purpose (does not affect the training result)

Reply:
ok